synapse/handlers/room.py

```diff
@@ -1896,6 +1982,16 @@ async def shutdown_room(
                 ratelimit=False,
             )
 
+            shutdown_response["new_room_id"] = new_room_id
```

Comment:
Before modifying `shutdown_response` further, should we copy it and return the copy?